code/game/objects/structures/vehicles/vehicle.dm

```diff
@@ -95,16 +96,56 @@
 			to_chat(user, "Need more welding fuel!")
 			return
 	else if(istype(W, /obj/item/key))
-		if(keytype)
-			to_chat(user, "Hold \the [W] in one of your hands while you drive \the [src].")
+		if(!heldkey)
+			if(keytype && mykey == W)
+				if(prob(95) && user.drop_item(W, src))
+					to_chat(user, "<span class='notice'>You insert \the [W] to \the [src]'s ignition and turn it.</span>")
+					user.visible_message("<span class='notice'>\The [src]'s engine roars to life!</span>")
+					playsound(get_turf(src), "sound/items/screwdriver.ogg", 10, 1) // find a better sound later
+					src.heldkey = W
+				else
+					to_chat(user, "<span class='warning'>You try to insert \the [W] to \the [src]'s ignition but you miss the slot!</span>")
```

Comment:
To what end? This is too rare to be reasonably considered balance, but too common to be a meme. It just seems like unnecessary annoyance.

Reply:
It's intended to be latter just for some fun with clumsy people later.

Reply:
Yes, but why make it apply to non-clumsy people?

Reply:
Don't make this apply to non-clumsy people/